tests/response/test_response_cookies.py

```diff
@@ -53,3 +53,48 @@ def test_method() -> None:
     with create_test_client(test_method) as client:
         response = client.get("/")
         assert response.headers["Set-Cookie"] == "test=123; Path=/; SameSite=lax"
+
+
+def test_response_cookie_documentation_only_not_rendering() -> None:
+    @get(
+        "/",
+        response_cookies=[
+            Cookie(
+                key="my-cookie",
+                description="my-cookie documentations",
+                documentation_only=True,
+            )
+        ],
+    )
+    def test_method() -> None:
+        return None
+
+    with create_test_client(test_method) as client:
+        response = client.get("/")
+        assert "Set-Cookie" not in response.headers
+
+
+def test_response_cookie_documentation_only_not_producing_second_header() -> None:
+    # this test ensures that https://github.com/starlite-api/starlite/issues/870
```

Comment:
This is good. We should do this habitually so people know why sometimes seemingly strange tests exists